packages/features/eventtypes/components/tabs/advanced/EventAdvancedTab.tsx

```diff
@@ -181,6 +185,19 @@ export const EventAdvancedTab = ({
       .filter((secondaryEmail) => !!secondaryEmail.emailVerified)
       .map((secondaryEmail) => ({ label: secondaryEmail.email, value: secondaryEmail.id })),
   ];
+
+  const removePlatformClientIdFromEmail = (email) => email.replace(/\+[^@]+/, "");
+
+  let userEmail = user?.email || "";
+
+  if (isPlatform) {
+    verifiedSecondaryEmails = verifiedSecondaryEmails.map((email) => ({
+      ...email,
+      label: removePlatformClientIdFromEmail(email.label),
+    }));
+    userEmail = removePlatformClientIdFromEmail(userEmail);
+  }
```

Comment:
pushing small change to this, we know the clientId and should only remove it not any other alias